templates/about.hbs

```diff
@@ -8,27 +8,28 @@
 
   <p>
   Docs.rs automatically checks crates released into
-  <a href="https://crates.io/">crates.io</a> and builds documentations. It is
-  using nightly release of Rust compiler.
+  <a href="https://crates.io/">crates.io</a> and builds documentations. It
+  uses nightly release of Rust compiler.
```

Comment:
nit: uses the nightly

Also, I'm thinking that it might be better to replace "checks" with "processes" or "documents" to make it clear that the purpose is to document crates.